### DISCUSSION TITLE
Unable to login with admin user after last update of Directus

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#9948] [Archive]
Since I migrated to the latest version of Directus, I'm unable to login with the admin user. I tried to configure mailing properly so that I can reset its password but it was a failure (tried with a Gmail account). Is there a command-line way (or another way) to reset admin password?

**Comment by mhdaouas** [comment]
Help please

**Comment by azrikahar** [comment]
> Since I migrated to the latest version of Directus, I'm unable to login with the admin user

Can you share what was the version you have migrated from? There were instances with particular versions on SQLite where the migration may cause the `role` column in `directus_users` to be empty, hence the login failure. Would you mind checking it via the database to see if this is the case here?

If the role column is empty for your admin user, you should be able to fill it in based on the ID in `directus_roles` table.

**Reply by mhdaouas** [reply]
I don't remember but I guess I migrated from the previous version (v9.0.0). The column contains the right value and the directus_roles table is fine too.

**Reply by azrikahar** [reply]
Hmm it's hard to tell what could be the issue in terms of Directus side of things with the limited information at hand, and there doesn't seem to be any similar reports for v9.0.0 to v9.0.1. It should be unlikely given how there shouldn't (AFAIK) be any migrations relating to that for v9.0.1. 

In regards to resetting admin password, one possible route would be generating it via `/utils/hash/generate` (reference: https://docs.directus.io/reference/system/utilities/#generate-a-hash) and replacing the password value from the DB side. It is still advisable to do a backup before attempting such action, or even any migrations going forward. 👍

**Reply by mhdaouas** [reply]
You saved me! It worked! Thank you very much for your great help!
